thorlcr/graph/thgraphslave.cpp

```diff
@@ -1485,6 +1493,13 @@ IThorResult *CSlaveGraph::getGlobalResult(CActivityBase &activity, IThorRowInter
     return result.getClear();
 }
 
+void serializeNullSlaveDoneGraph(MemoryBuffer &mb, graph_id gid)
```

Comment:
This doesn't seem to be used - probably because fix changed.

Reply:
I meant to remove.